tactics/redexpr.ml

```diff
@@ -209,18 +208,14 @@ let out_with_occurrences (occs,c) =
 
 let e_red f env evm c = evm, f env evm c
 
-let head_style = false (* Turn to true to have a semantics where simpl
-   only reduce at the head when an evaluable reference is given, e.g.
-   2+n would just reduce to S(1+n) instead of S(S(n)) *)
-
-let contextualize f g = function
+let contextuallize f = function
```

Comment:
Trivial question but why the double 'l' ? AFAICT this variant doesn't appear in dictionaries

Reply:
I have no idea how this change appeared

Reply:
changed